sql/partialize_finalize.sql

```diff
@@ -8,24 +8,23 @@
 -- parallel-safe.
 
 CREATE OR REPLACE FUNCTION _timescaledb_internal.partialize_agg(arg ANYELEMENT)
-RETURNS BYTEA AS '@MODULE_PATHNAME@', 'ts_partialize_agg' LANGUAGE C STABLE PARALLEL SAFE;
+RETURNS BYTEA AS '@MODULE_PATHNAME@', 'ts_partialize_agg' LANGUAGE C VOLATILE;
```

Comment:
Even volatile? I'd think it should be immutable, it's a pure function.

Reply:
Yeah, why not keep STABLE and just remove parallel safe?

Reply:
Is it because some aggs can be volatile themselves?

Reply:
> Is it because some aggs can be volatile themselves?

Postgres check the volatility of the entire expression tree, so this shouldn't matter.

Reply:
I didn't thought too much about it because the idea here is to REVERT to the previous state.

Reply:
Yeah, I also think that this function is essentially just a planner hint and not actually executed.